### DISCUSSION TITLE
Project Maintainers Chat

### DISCUSSION THREAD

**Discussion by NicoleAbuhakmeh-zz** [dotnet-foundation/Home#13] [Project Help]
Leave items here for the community to jump in where you need it!

**Comment by michael-hawker** [comment]
Be sure to check out the monthly maintainer meeting [thread here](https://github.com/dotnet-foundation/Home/discussions/73)

And their announcements and working [repo is here](https://github.com/dotnet-foundation/wg-maintainers).
